integration_tests/integration_tests/tests/data_generator.py

```diff
@@ -3,10 +3,11 @@
 DATE_FORMAT = "%Y-%m-%d %H:%M:%S"
 
 
-def generate_dates(base_date, period="days", count=1, days_back=31):
+def generate_dates(base_date, step=None, days_back=31):
```

Comment:
This will be a breaking change after I merge my tests.

Reply:
ill fix it